modules/ROOT/pages/dataweave-type-reuse-module-loader.adoc

```diff
@@ -0,0 +1,432 @@
+= Reusing XML Schema Module Loader
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
```

Comment:
Regarding the file name `dataweave-type-reuse-module-loader.adoc`, since this page is about loading an XML schema, should it follow the naming pattern for the JSON schema loader (`dataweave-type-reuse-json-schema.adoc`) to become  `dataweave-type-reuse-xml-schema.adoc`?